pages-css/products-page-css/layout-handeler.css

```diff
@@ -0,0 +1,130 @@
+@import url("/pages-css/compo-css/navbar.css");
+@import url("/pages-css/products-page-css/page-style.css");
+@import url("https://fonts.googleapis.com/css2?family=Orienta&family=Quicksand&display=swap");
+:root {
+    --color-footer-background: #cecece;
+    --color-red: #fc0101;
+    --color-white: #ffffff;
+    --color-dark: #050505;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Orienta', sans-serif;
+    font-family: 'Quicksand', sans-serif;
+    scroll-behavior: smooth;
+}
+
+
+/* page container */
+
+.page-layout-container {
+    display: grid;
+    gap: 5px;
+    grid-template-columns: 1fr 3fr 2fr;
+    column-gap: 5px;
+    row-gap: 5px;
+    grid-template-areas: "header header header " "sidebar section-1 section-1 " "footer footer footer ";
+    padding: 5px;
+    padding-left: 1rem;
+    padding-right: 1rem;
+}
+
+
+/* *** */
+
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    grid-area: header;
+    height: 3.7rem;
+}
+
+.section-1 {
+    grid-area: section-1;
+    padding-left: 1.6rem;
+    padding-right: 1.6rem;
+}
+
+.sidebar {
+    display: flex;
+    flex-direction: column;
+    width: 8rem;
+    padding-left: 1.3rem;
+    line-height: 1.5rem;
+}
+
+.spacer {
+    padding: 1rem;
+}
+
+.footer {
+    background: var(--color-footer-background);
+    color: var(--color-dark);
+    grid-area: footer;
+    text-align: center;
+    height: fit-content;
+    padding: 2rem;
+}
+
+.odd-color-text {
+    color: var(--color-red);
+}
+
+ul li {
+    display: inline;
+    padding: 15px;
+    list-style-type: none;
+    color: var(--color-dark);
+}
+
+a {
+    font-weight: bold;
+    font-size: 1rem;
+    color: var(--color-dark);
+    text-decoration: none;
+}
+
+.highlight-text {
+    font-weight: bolder;
+    color: var(--color-red);
+    background: var(--color-white);
+}
+
+
+/* *** */
+
+
+/* section-1 */
+
+.home-page-content-1>p {
+    font-size: 15px;
+    padding-top: 10px;
```

Comment:
Use rem here

Reply:
Someone earlier suggested use px below 1 rem, so

Reply:
ohh so you haven't changed the root font size to 10px.Then i think its fine